src/sub-blocks/BasicCard/__stories__/BasicCard.stories.tsx

```diff
@@ -23,28 +23,14 @@ const DefaultTemplate: StoryFn<BasicCardProps> = (args) => (
     </div>
 );
 
-const WithIconTopTemplate: StoryFn<BasicCardProps> = (args) => (
+const WithIconTemplate: StoryFn<BasicCardProps> = (args) => (
```

Comment:
Can we left here only two cards with specific headers?
you can see the example of borders 
https://preview.gravity-ui.com/page-constructor/481/?path=/story/components-cards-basiccard--with-border

Reply:
agree